test/postgrex_test.exs

```diff
@@ -1,30 +1,34 @@
 defmodule PostgrexTest do
   use ExUnit.Case, async: false
-  import ExUnit.CaptureLog
 
-  test "start_link/2 sets search path" do
-    # valid argument
-    search_path = ["public", "extension"]
-    {:ok, conn} = Postgrex.start_link(database: "postgrex_test", search_path: search_path)
-    %{rows: [[result]]} = Postgrex.query!(conn, "show search_path", [])
-    assert result == Enum.join(search_path, ", ")
+  # This test fails due to a bug betweem Elixir and Erlang in earlier versions of Elixir.
+  if Version.match?(Version.parse!(System.version()), Version.parse_requirement!(">= 1.17.2")) do
```

Comment:
We have a simplified version of this in Ecto:

`if Version.match?(System.version(), ">= 1.13.0-dev") `